### PR TITLE
Move console to separate chart

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Move console to separate chart

**Related issue(s)**

### CODE REVIEW

File: resources/console/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Core
```
Comment (Wawrzyn321) [inline resources/console/README.md:1]: Shouldn't it be about Console?

Comment (dariadomagala) [reply resources/console/README.md:1]: It should :) I'll change it now

**Comment by Wawrzyn321** [comment]
I've installed it locally and Console can be accessed under `console.kyma.local`, but installation message doesn't show the address:
image

**Comment by Wawrzyn321** [comment]
image
`Diagnostics` are missing on clean install.

**Comment by dariadomagala** [comment]
The "console not installed" issue will have to be changed in kyma-cli repo

**Comment by dariadomagala** [comment]
@Wawrzyn321 The "Diagnostic" category is not installed by default, you would have to enable it to have it locally on your minikube.

File: resources/console/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Console
+
+## Overview
+
+The Console is a web-based administrative UI for Kyma. It allows to administer the Kyma functionality and manage the basic Kubernetes resources. Console chart consists of the following sub-charts:
```
Comment (bszwarc) [inline resources/console/README.md:5]:
```suggestion
The Console is a web-based administrative UI for Kyma. It allows administering the Kyma functionality and managing basic Kubernetes resources. The Console chart consists of the following sub-charts:
```

**Comment by sjanota** [comment]
@akucharska It's because [changes in CLI](https://github.com/kyma-project/cli/pull/467) can only be merged after this PR.

File: resources/console/values.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+console:
+  cluster:
+    headerLogoUrl: "assets/logo.svg"
+    headerTitle: ""
+    faviconUrl: "favicon.ico"
+
+global:
+  isLocalEnv: false
+  knative:
+    false
```
Comment (Demonsthere) [inline resources/console/values.yaml:14]:
```suggestion
  knative: false
```

**Pull request merged**